jbpm-services/jbpm-kie-services/pom.xml

```diff
@@ -93,7 +93,11 @@
         </exclusion>
       </exclusions>
     </dependency>  
-  
+    <dependency>
+      <groupId>org.uberfire</groupId>
```

Comment:
We are introducing an uberfire dependency in the core?  This seems strange to me, UF / AppFormer is all / only about our webtooling?